technical-design/uml/sequence-fetchProtectedData.md

```diff
@@ -9,16 +9,28 @@ sequenceDiagram
         participant SDK as @iexec/dataprotector
     end
 
-    participant DPSC as DataProtector SC
     participant DPSG as DataProtector Subgraph
+    participant CCSC as ContentCreator SC
+    participant DPSC as DataProtector SC
 
-    DPSG --) DPSC: observe and index new Protected Data
+    DPSG --) DPSC: observe and index events
+
+    CCSC --) DPSC: observe and index events
 
     User -) SDK: fetchProtectedData<br>(owner optional filter,<br>dataSchema optional filter)
 
-    SDK ->> DPSG: query protected data matching filters
+    SDK ->> SDK: check if the CC_Contract <br> owns the protectedData
```

Comment:
Comment il fait le SDK pour savoir ça lui-même ?

<img width="150" alt="Capture d’écran 2023-12-22 à 16 22 01" src="https://github.com/iExecBlockchainComputing/dataprotector-sdk/assets/4280765/977c9275-e868-4696-b7ff-3cc6108bb14a">

Ou j'ai mal compris. Est-ce qu'ici c'est pas plutôt : "Checks if the owner filter is the CC Contract" ?

Reply:
oui tout a fait pardon petite coquille

Reply:
Y'a un appel à cette fonction du iexec-sdk que j'ai masqué : https://github.com/iExecBlockchainComputing/iexec-sdk/blob/master/docs/classes/IExecDatasetModule.md#showuserdataset
Qui elle même fait un call au Poco. Je peux le faire apparaitre effectivement 🤷🏻‍♂️